From 4ae6c61d1ab721672ecab5a4ae3949a3e05930cf Mon Sep 17 00:00:00 2001
From: Peter Rutschmann <peter.rutschmann@bbw.ch>
Date: Wed, 12 Apr 2023 13:30:48 +0200
Subject: [PATCH 08/27] The input component shall be used.

---
 src/formelrad/Formelrad.js | 20 +++++++++++---------
 1 file changed, 11 insertions(+), 9 deletions(-)

diff --git a/src/formelrad/Formelrad.js b/src/formelrad/Formelrad.js
index 5f149a6..994e2b5 100644
--- a/src/formelrad/Formelrad.js
+++ b/src/formelrad/Formelrad.js
@@ -1,20 +1,22 @@
+import {useState} from "react";
 import formelrad from "../image/formelradelektronik.gif";
+import InputField from "../formular/InputField";
 
 export default function Formelrad() {
+    const [values, setValues] = useState({
+        u: 10,
+        i: 2,
+        r: ""
+    })
+
     return (
         <>
             <h2>Formelrad</h2>
             <img src={formelrad} width="200" alt="Formelrad"/>
             <form>
-                <div>
-                    <label>Spannung</label><input></input>
-                </div>
-                <div>
-                    <label>Strom</label><input></input>
-                </div>
-                <div>
-                    <label>Widerstand</label><input></input>
-                </div>
+                <InputField color={"black"} value={values.u} label="Spannung" handleChange={e => {setValues(values => ({...values, u: e.target.value}))}} />
+                <InputField color={"black"} value={values.i} label="Stromstaerke" handleChange={e => {setValues(values => ({...values, i: e.target.value}))}} />
+                <InputField color={"black"} value={values.r} label="Widerstand" handleChange={e => {setValues(values => ({...values, r: e.target.value}))}} />
                 <button type="submit">Calculate</button>
             </form>
         </>
-- 
2.30.0.windows.1

